pkg/cmd/cli/config/smart_merge.go

```diff
@@ -0,0 +1,157 @@
+package config
```

Comment:
Is this a straight copy of: https://github.com/openshift/origin/blob/master/pkg/cmd/experimental/login/smart_merge.go?  Need to clean up the original?